packages/cactus-plugin-ledger-connector-quorum/src/test/resources/network-files/member1/nodekey

```diff
@@ -0,0 +1 @@
+b9a4bd1539c15bcc83fa9078fe89200b6e9e802ae992f13cd83c853f16e8bed4
```

Comment:
@travis-payne Where are these being used? (meaning the whole network-files directory)

Reply:
As below, not all the files are being used, but figured I'd leave them all there in case someone wants to expand on functionality  that I might not foresee right now. Can remove if it useless though.

https://github.com/hyperledger/cactus/blob/24e3b0dd08fff550cbba7785566eea19daee1edd/packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v2.3.0-deploy-contract-from-json-private.test.ts#L201

Reply:
@travis-payne Only saw this by accident just now, please tag me otherwise it doesn't show up on my mentions feed.
I'd throw everything out that's not being used because we don't know when that future date will come and if these files will be outdated by then which would also lead to some additional confusion.